common/app/views/fragments/bashers/KeepItInTheGround.scala.html

```diff
@@ -0,0 +1,22 @@
+@()(implicit request: RequestHeader)
+
+<div class="gs-container">
+    <div class="keep-it-in-the-ground">
+        <a class="keep-it-in-the-ground__anchor" href="http://www.theguardian.com/environment/2015/oct/05/a-story-of-hope-the-guardian-launches-phase-two-of-its-climate-change-campaign"></a>
+        <div class="keep-it-in-the-ground__image"></div>
+        <div class="keep-it-in-the-ground__content">
+            <h2 class="keep-it-in-the-ground__headline">Join the Guardian’s campaign and help spread a message of hope that the world can stop climate change<br /> <strong>Join us
+                <div class="keep-it-in-the-ground__button">
```

Comment:
Is this a button? (I presume so from the class.) If so, see `<button>` which has lovely semantics :wink: